packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/EthERC20Bridge.sol

```diff
@@ -203,7 +223,7 @@ contract EthERC20Bridge is L1Buddy {
 
         bytes memory data = abi.encodeWithSelector(selector, erc20, destination, amount, decimals);
         return
-            inbox.createRetryableTicket(
```

Comment:
Add event

Reply:
this one one already has DepositERC20 and DepositERC777 where it gets called